test/e2e/vault-decryption-chrome.spec.ts

```diff
@@ -218,6 +219,7 @@ describe('Vault Decryptor Page', function () {
     await withFixtures(
       {
         disableServerMochaToBackground: true,
+        title: this.test?.fullTitle(),
```

Comment:
unrelated, but I spot this was missing on the vault spec, and as it's a small tweak I included in this same PR